templates/aws-stack.yml

```diff
@@ -1016,6 +1051,24 @@ Resources:
       AutoScalingReplacingUpdate:
         WillReplace: true
 
+  AutoScalingStartSchedule:
+    Type: AWS::AutoScaling::ScheduledAction
+    Condition: HasStartSchedule
+    Properties: 
+      AutoScalingGroupName: !Ref AgentAutoScaleGroup
+      MinSize: !Ref ScheduleMinSize
```

Comment:
I wonder if `ScheduleMinSize ` should get the same conditional treatment as `ScheduleMaxSize`.
Some teams might want to adjust their MaxSize on a schedule but leave MinSize zero regardless.
Would that make sense?

Reply:
I can see that use case - I'll add it in.